discord/state.py

```diff
@@ -814,6 +814,42 @@ def parse_message_reaction_remove_emoji(self, data) -> None:
                 if reaction:
                     self.dispatch("reaction_clear_emoji", reaction)
 
+    def parse_message_poll_vote_add(self, data) -> None:
```

Comment:
maybe we could cache polls and results on cached messages? just an idea

Reply:
Seems like an idea